dcos/dcos-ui
# [spike day] DCOS-21203: Volumes size label is uppercase and shouldn't be

Change the text to volume size to not be uppercase.

Closes https://jira.mesosphere.com/browse/DCOS-21203

## Testing
0. Checkout https://github.com/mesosphere/dcos-ui-plugins-private/pull/899
1. Go to services
2. Create a new single-container service
3. Go to the Volumes tab
4. Select different options from the dropdown and verify that Size is "Size" and not "SIZE" everywhere
5. Verify the same for a multi-container service 

## Trade-offs
None.

## Dependencies
https://github.com/mesosphere/dcos-ui-plugins-private/pull/899

## Screenshots

![screenshot from 2018-12-21 15-19-04](https://user-images.githubusercontent.com/40791275/50344606-5db39800-0534-11e9-9640-bc5f9df3ba2e.png)

![screenshot from 2018-12-21 15-20-00](https://user-images.githubusercontent.com/40791275/50344612-6310e280-0534-11e9-80b5-c250c1efe4cc.png)

![screenshot from 2018-12-21 15-20-08](https://user-images.githubusercontent.com/40791275/50344613-64daa600-0534-11e9-962c-2135196f4adb.png)
